api/requirements.txt

```diff
@@ -6,6 +6,9 @@ flask-discord~=0.1.64
 
 # database
 flask-sqlalchemy~=2.5.1
+PyMySQL~=1.0.2
+psycopg2-binary~=2.8.6
```

Comment:
Do we really still need the binary version?

Reply:
Nope I tried it without and it works.